src/Data/Container/Indexed/Relation/Binary/Pointwise/Properties.agda

```diff
@@ -0,0 +1,48 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of pointwise equality for indexed containers
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Container.Indexed.Relation.Binary.Pointwise.Properties where
+
+open import Axiom.Extensionality.Propositional
+open import Data.Container.Indexed.Core
+open import Data.Container.Indexed.Relation.Binary.Pointwise
+open import Data.Product using (_,_; Σ-syntax; -,_)
+open import Level using (Level; _⊔_)
+open import Relation.Binary
+open import Relation.Binary.PropositionalEquality as P
+  using (_≡_; subst; cong)
+
+private variable
+  ℓᵉ ℓᵖ ℓˢ ℓˣ : Level
+  I O : Set _
+
+module _ (C : Container I O ℓˢ ℓᵖ) {X : I → Set ℓˣ}
+         (R : (i : I) → Rel (X i) ℓᵉ)
+         {o : O}
+         where
+
+  refl : (∀ i → Reflexive (R i)) → Reflexive (Pointwise C R o)
```

Comment:
(Again modulo an implicit argument) we have the type of indexed reflexivity/symmetry etc. already defined:

https://github.com/agda/agda-stdlib/blob/358ad8bf673f0b292f5e5e86b50f0f6ec1d245cd/src/Relation/Binary/Indexed/Homogeneous/Definitions.agda#L36